app/helpers/proposals_helper.rb

```diff
@@ -11,4 +11,12 @@ def datespan_header(start_date, end_date)
       end
     end
   end
+
+  def diff_val(val)
+    if val.is_a?(Numeric)
+      format('%.2f', val)
+    else
+      val.inspect
+    end
+  end
```

Comment:
Would it make sense to reuse/repurpose `fields_for_display`? Ideally there'd be more meta data so we'd know field x is a dollar amount, etc.